spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    # sequence(:email) { |n| "user#{n}@gmail.com"}からの書き換え
+    sequence(:email, "user1@sample.com")
```

Comment:
存在しないことが保証されているドメインである`example.com`を使うようにしましょう。
```suggestion
    sequence(:email, "user1@example.com")
```

Reply:
了解しました！